graph/multi/directed.go

```diff
@@ -247,26 +248,11 @@ func (g *DirectedGraph) Lines(uid, vid int64) []graph.Line {
 	for _, l := range edge {
 		lines = append(lines, l)
 	}
-	return lines
+	return iterator.NewOrderedLines(lines)
 }
 
 // HasEdgeFromTo returns whether an edge exists in the graph from u to v.
 func (g *DirectedGraph) HasEdgeFromTo(uid, vid int64) bool {
 	_, ok := g.from[uid][vid]
 	return ok
 }
-
-// Degree returns the in+out degree of n in g.
```

Comment:
Was this removed as the functionality is now provided by iterators? If so, how.

Reply:
`id := n.ID(); g.From(id).Len()+g.To(id).Len()` gives us this. This is currently more costly, but will be less so later (not quite down to what it is right now, but I think worth it because of the readability; there are many ways to interpret degree, and a method name of `Degree` does not say which, the code above does.